k8s_snapshots/backends/google.py

```diff
@@ -239,11 +279,19 @@ def get_snapshot_status(
     gcloud = get_gcloud(ctx)
 
     # First, check the operation state
-    operation = gcloud.zoneOperations().get(
-        project=get_project_id(ctx),
-        zone=snapshot_identifier['zone'],
-        operation=snapshot_identifier['operation_name']
-    ).execute()
+
+    if "region" in snapshot_identifier:
```

Comment:
same deal as my comment above

Reply:
Same as my comment above, the function calls are different, requiring either region or zone be set depending on which type of PVC it is

Reply:
this is what happens when i PR review in the evening...